backend/app/adapter/sqldb/short_link_integration_test.go

```diff
@@ -162,7 +162,7 @@ func TestShortLinkSql_GetByAlias(t *testing.T) {
 					insertShortLinkTableRows(t, sqlDB, testCase.tableRows)
 
 					shortLinkRepo := sqldb.NewShortLinkSql(sqlDB)
-					shortLink, err := shortLinkRepo.GetByAlias(testCase.alias)
+					shortLink, err := shortLinkRepo.GetShortLinkByAlias(testCase.alias)
```

Comment:
Need to rename the test case function from `TestShortLinkSql_GetByAlias ` to `TestShortLinkSql_ GetShortLinkByAlias`.